hack/docker-push.sh

```diff
@@ -93,8 +101,15 @@ echo "Building and pushing container images."
 # The use of "registry" as the buildx output type below instructs
 # Docker to push the image
 
+if [[ $NOT_PUSH_IMAGE = true ]]; then
```

Comment:
let's make it consistent with line 54 like
`[[ "$triggeredBy" == "heads" ]];`

Reply:
This check is removed.